src/pages/index.js

```diff
@@ -1,75 +1,182 @@
 import "../pages/index.css";
 import Card from "../components/Card.js";
-import { enableValidation, formValidators } from "../components/FormValidator.js";
 import Section from "../components/Section.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 import UserInfo from "../components/UserInfo.js";
-import { initialCards, validationSettings } from "../utils/Constants.js";
-
- // Elements
- const profileEditButton = document.querySelector("#profile-edit-button");
- const addNewCardButton = document.querySelector(".profile__add-button");
-
- // Forms
- const profileForm = document.forms['profile-form'];
- const cardForm = document.forms['card-form'];
-
- // Instances
- const userInfo = new UserInfo({ nameSelector: ".profile__title", descriptionSelector: ".profile__description" });
- const popupWithImage = new PopupWithImage("#modal-preview-picture");
- popupWithImage.setEventListeners();
- const popupWithFormProfile = new PopupWithForm("#profile-edit-modal", handleProfileFormSubmit);
- popupWithFormProfile.setEventListeners();
- const popupWithFormAddCard = new PopupWithForm("#add-card-modal", handleAddCardFormSubmit);
- popupWithFormAddCard.setEventListeners();
-
- // Function to create card
- function createCard(cardData) {
-   const card = new Card(cardData, "#card-template", () => popupWithImage.open(cardData));
-   return card.getView();
- }
-
- // Function to handle profile form submit
- function handleProfileFormSubmit(formData) {
-   userInfo.setUserInfo({
-     name: formData.name,
-     description: formData.description
-   });
-   popupWithFormProfile.close();
- }
-
- // Function to handle add card form submit
- function handleAddCardFormSubmit(formData) {
-   const cardElement = createCard({ name: formData.title, link: formData.url });
-   cardSection.addItem(cardElement);
-   popupWithFormAddCard.close();
-   cardForm.reset(); // Clear the form fields only after submission
-   formValidators['card-form'].disableButton(); // Disable submit button after form reset
```

Comment:
```js

   cardForm.reset(); // Clear the form fields only after submission
   formValidators['card-form'].disableButton(); // Disable submit button after form reset
```

You had this code last time. Please, add it to the block `then` after submitting